internal/cmd/schema-registry/command_schema_describe.go

```diff
@@ -223,7 +224,18 @@ func printSchema(cmd *cobra.Command, schemaID int64, schema string, sType string
 	if sType != "" {
 		utils.Println(cmd, "Type: "+sType)
 	}
-	utils.Println(cmd, "Schema: "+schema)
+
+	if sType == "PROTOBUF" {
+		utils.Println(cmd, "Schema:\n"+schema)
+	} else {
+		var jsonBuffer bytes.Buffer
+		if err := json.Indent(&jsonBuffer, []byte(schema), "", "    "); err != nil {
+			return err
+		}
+		schemaString := jsonBuffer.String()
+		utils.Println(cmd, "Schema:\n"+schemaString)
+	}
```

Comment:
Looks like we can extract the `utils.Println()` calls, since they're duplicated. It's also unclear what `sType` means; can we rename it?
```suggestion
	switch sType {
	case "PROTOBUF":
		utils.Println(cmd, "Schema:")
		utils.Println(cmd, schema)
	default:
		var jsonBuffer bytes.Buffer
		if err := json.Indent(&jsonBuffer, []byte(schema), "", "    "); err != nil {
			return err
		}
		schemaString := jsonBuffer.String()
		utils.Println(cmd, "Schema:")
		utils.Println(schemaString)
	}
```